be/src/olap/rowset/segment_v2/column_reader.h

```diff
@@ -188,5 +191,39 @@ class FileColumnIterator : public ColumnIterator {
     rowid_t _current_rowid = 0;
 };
 
+// This iterator is used to read default value column
+class DefaultValueColumnIterator : public ColumnIterator {
+public:
+    DefaultValueColumnIterator(const TabletColumn& tablet_column)
+        : _tablet_column(tablet_column) { }
```

Comment:
initialize member fields `_is_default_value_null`, `_default_value`, and `_value_size`

Reply:
ok, I missed them